.github/workflows/main.yml

```diff
@@ -58,6 +63,8 @@ jobs:
 
     - name: Test
       run: |
+        export OMP_NUM_THREADS=1
```

Comment:
Can you create an issue that reports the failure so we can try to fix this later?

For a bit of explainations: openMP is a library that is used to make some parallel computations, typically for numpy operations but also probably for `snapml`. Default behavior is to use all cores to run the operations in parallel.

By setting `OMP_NUM_THREADS=1`, we bypass the default behavior and requires sequential computations. So if setting this solves the problem, this is most probably an issue of concurrency in `snapml` code.

Reply:
Yes, that's done now (#20)
Ok, thanks for the explanation